src/main/java/ua/edu/internship/interview/web/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,57 @@
+package ua.edu.internship.interview.web.handler;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import ua.edu.internship.interview.service.utils.exceptions.BaseException;
+
+import java.util.HashMap;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.HashMap'
<!-- __reviewdog__:ChBmZDZkYzBlM2I3YTkwMjM2EgpjaGVja3N0eWxl -->